dips/dip-7.md

```diff
@@ -14,40 +14,56 @@ This DIP outlines the process by which technical working groups (TWG’s) are fo
+
+In the event that additional resources are needed, the facilitator should request those items from the TSC.
+
 ## Decision Making in a TWG
-The TWG facilitator owns the agenda, meetings, and organization of the TWG, but is not a final decision maker. They are there to drive and facilitate discussion, and help allocate resources so that recommendations, reports, or draft DIPs can be created. They are also responsible for providing a bi-weekly update to the association on the progress the TWG is making and any associated documentation being produced. Bi-weekly updates should be posted in the Diem portal and distributed via email/slack to all Association members.
+The TWG facilitator owns the agenda, meetings, and organization of the TWG, but is not a final decision maker. They are there to drive and facilitate discussion, and help allocate resources so that recommendations, reports, or draft DIPs can be created. They are also responsible for providing updates to the association and community on the progress the TWG is making and any associated documentation being produced. Bi-weekly updates are suggested and should be posted in the Diem portal, distributed via email/slack to all Association members, and circulated in public forums as necessary.
 
-If there are material changes to the TWG’s scope, timeline, or expected output, a revised proposal must be submitted to the TSC for re-approval.
+If there are highly material changes to the TWG’s scope, a revised proposal must be submitted to the TSC for re-approval.
 
-Any TWG member can propose any topic or idea within the scope of the TWG up for discussion with the group. These discussions should be documented, and as the group comes to agreement on how to proceed, there should be clear ownership among TWG members for drafting, editing, obtaining feedback, and finalizing the group’s working product. There does not need to be total agreement in the TWG on the final output, but a supermajority (67%+) vote in favor is required before proceeding.
+Any TWG member can propose any topic or idea within the scope of the TWG up for discussion with the group. These discussions should be documented, and as the group comes to agreement on how to proceed, there should be clear ownership among TWG members for drafting, editing, obtaining feedback, and finalizing the group’s working product. A TWG can have any number of outputs.There must be rough consensus in the TWG on each final work product before proceeding. 
 
-Once the TWGs final output is ready and the group votes to proceed, it must be shared with the TSC for review and edits. After the editing process, the TSC must approve the output before it can be circulated within the Association.
+Once rough consensus is found, a work product must be shared with the TSC for review and edits. After the editing process, the TSC must approve the output via a majority vote. The work product can then be officially shared with the Association as final.
```

Comment:
Again, this is at the discretion of the Lead Maintainer -- which artifact needs to be shared with the TSC and in what form.